src/main/java/org/kohsuke/github/GitHubInteractiveObject.java

```diff
@@ -20,4 +20,8 @@
     GitHubInteractiveObject(GitHub root) {
         this.root = root;
     }
+
+    GitHub getRoot() {
+        return root;
+    }
```

Comment:
Added this internal method for tests to use, in preparation changing `root` to private.